content/guides/events-api-connection/warning.md

```diff
@@ -0,0 +1,18 @@
+# Limitations
```

Comment:
We should probably only add a link to the existing docs https://api.akeneo.com/events-documentation/limits-and-scalability.html
And to be honest I don't understand why we don't add theses limits to the existing docs